images/win/scripts/Installers/Install-Nuget.ps1

```diff
@@ -0,0 +1,17 @@
+################################################################################
+##  File:  Install-Nuget.ps1
+##  Desc:  Install NuGet.CommandLine
+################################################################################
+
+Choco-Install -PackageName NuGet.CommandLine
+
+$env:Path = $env:Path + ";$env:ChocolateyInstall\bin"
```

Comment:
I believe this line should already been in PATH after installation of Choco in previous steps